src/lib/jsonParseV2/parseSubscription.cpp

```diff
@@ -48,40 +49,43 @@ static std::string parseEntitiesVector(ConnectionInfo* ciP, EntityIdVector* eivP
 static std::string parseNotifyConditionVector(ConnectionInfo* ciP, SubscribeContextRequest* scrP, const Value& condition);
 
 
+
 /* ****************************************************************************
 *
 * parseSubscription -
 *
 */
-std::string parseSubscription(ConnectionInfo* ciP, ParseData* parseDataP, bool update)
+std::string parseSubscription(ConnectionInfo* ciP, ParseData* parseDataP, JsonDelayedRelease* releaseP, bool update)
 {
-  Document                 document;
-  SubscribeContextRequest* destination;
+  Document                  document;
+  SubscribeContextRequest*  destination;
 
   document.Parse(ciP->payload);
 
   if (update)
   {
-    destination = &parseDataP->ucsr.res;
+    destination     = &parseDataP->ucsr.res;
+    releaseP->ucsrP = &parseDataP->ucsr.res;
```

Comment:
releaseP->ucsrP is set to point to &parseDataP->ucsr.res, so that it will be released after usage